content/id/docs/tasks/administer-cluster/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Mengelola sebuah Klaster"
```

Comment:
It is a good point! We need an index for "Mengelola Sebuah Klaster" but I think we don't need double quotes here.
```suggestion
title: Mengelola Sebuah Klaster
```

Reply:
Since it doesn't have any index yet I think it's better to just add it!
Resolved in e323b8c5920d6ae02c050306495e784b8e19ad9f